src/interface.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeEvent, MouseEvent } from "react";
+
+export interface XploresAIType {
+  handleInputChange: (e: ChangeEvent<HTMLTextAreaElement>) => void;
+  handleSubmit: (e: MouseEvent<HTMLButtonElement>) => Promise<void>;
+  handleFileUpload: (e: FileUploadEvent) => void;
+  value: string;
+  loading: Boolean;
```

Comment:
Not sure why are you using Boolean as it is not a type but an object. 
```suggestion
  loading: boolean;
```